StaticWebsite/models.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
i also think you're missing the events models to dynamically allow us to add events and have them show on the website